src/components/BtnNumbers.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable quotes, jsx-quotes, react/prop-types */
+import React from 'react';
+
+const BtnNumbers = ({ item }) => (
+  <div>
+    <button className='btn btn-secondary buttons' type='button'>
+      {item}
+    </button>
+  </div>
+);
+
+export default BtnNumbers;
```

Comment:
- [ ] It is required that all the components should be class-based components. Kindly change this component from functional to class-based component.

![project-rqt](https://user-images.githubusercontent.com/26597432/211640363-fb2746ca-5415-412d-9641-8af03e1d6c91.png)